streamlit-ui/requirements.txt

```diff
@@ -0,0 +1,9 @@
+./generated
+auth0-python
+pandas
+pyjwt
```

Comment:
Protip: you probably want to pin/freeze (`pip freeze`) the versions of these!

Reply:
How do I know? I didn't do it for `documentation` until things started breaking 🤣